src/components/date-range-picker/date-localization.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { formatDate } from '@angular/common';
+import { DateLocalizationFormatter } from './date-localization-formatter.type';
+
+@Pipe({
+    name: 'dateLocale'
```

Comment:
This should probably be something like `formatDate` or `formattedDate`, i.e. describing the outcome of the pipe. Also we have a separate subdirectory for custom pipe components under `src` which could be useful in case we need this functionality in other date-based components in the future.